{"name": "not authenticated user -- @1.14 cases", "status": "passed", "steps": [{"name": "Given alien makes post request: /reconciliation/status/", "status": "passed", "start": 1611335753560, "stop": 1611335753585}, {"name": "Then check response of request: Authentication credentials were not provided. is True", "status": "passed", "start": 1611335753585, "stop": 1611335753585}], "parameters": [{"name": "url", "value": "/reconciliation/status/"}, {"name": "method", "value": "post"}, {"name": "bool", "value": "True"}], "start": 1611335753557, "stop": 1611335753585, "uuid": "619ba2f9-20f2-430c-b169-bdd88501c768", "historyId": "8549902b6bce57d707aa81fd9c5b34d9", "fullName": "features/II-Reconciliation/III-reconciliation_access.feature:not authenticated user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}